tvb/basic/logger/builder.py

```diff
@@ -78,7 +78,7 @@ def build_logger(self, parent_module):
         return logger
 
     def set_loggers_level(self, level):
-        for logger in self._loggers.values():
+        for logger in list(self._loggers.values()):
```

Comment:
you should not need a `list` here since `list` would only work on an iterable anyway